staging/src/k8s.io/sample-controller/.gitignore

```diff
@@ -0,0 +1 @@
+sample-controller
```

Comment:
do we have the same for all binaries?

Reply:
We do have examples of binaries in the .gitignore:

- https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/pod-security-admission/.gitignore
- https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/.gitignore
- https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/mounter/.gitignore

but I'm sure there are more directories that produce binaries that do not have the ignore. One step at a time

Reply:
oh, you are saying we naturally produce that binary (during e2e?)